config/rbac/service_account.yaml

```diff
@@ -4,4 +4,8 @@ metadata:
   name: operator
   namespace: system
   labels:
+    app.kubernetes.io/managed-by: "ssp-operator"
+    app.kubernetes.io/version: "0.0.0"
+    app.kubernetes.io/component: "ssp-operator"
+    app.kubernetes.io/part-of: "hyperconverged-cluster"
```

Comment:
This value `hyperconverged-cluster` should probably be set by HCO or OLM.

Reply:
Okay, how we can achieve it? by changing some HCO code to set it? This `service_account.yaml` is not part of some code, where we can use something like `WithAppLabels()`.

Reply:
Hi @orenc1 @tiraboschi @nunnatsa, do you know, by chance, if these labels can be set by OLM? Is it required by [1].

[1] https://issues.redhat.com/browse/CNV-9097

Reply:
yes, HCO is setting the labels on all underlying components when constructing the CSV:
see here:
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/901f0dd11ca46c928033dd8987edecb0c601b2ea/tools/csv-merger/csv-merger.go#L541C43-L541C64

Reply:
Okay, I see. Question is, why the service-account here [1] doesn't contain any labels? also the same question about other resources that HCO deploys, e.g. [2].

[1] https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/deploy/service_account.yaml#L42-L48
[2] https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/deploy/operator.yaml#L498-L500